packages/scripts/src/profound/getAndProcessAnswers.ts

```diff
@@ -144,21 +137,24 @@ const model: ModelConfig = {
 
 export const main = async (startDateArg?: string, endDateArg?: string) => {
   // START setup
-  // Parse optional startDate and endDate from command line arguments
-  let start: Date, end: Date;
-  if (startDateArg) {
-    const startDate = new Date(startDateArg);
-    const endDate = endDateArg ? new Date(endDateArg) : startDate;
-    ({ start, end } = getFullDayRange(startDate, endDate));
-  } else {
-    // Default to yesterday
-    const now = new Date();
-    const utcYear = now.getUTCFullYear();
-    const utcMonth = now.getUTCMonth();
-    const utcDate = now.getUTCDate();
-    const yesterday = new Date(Date.UTC(utcYear, utcMonth, utcDate - 1));
-    ({ start, end } = getFullDayRange(yesterday, yesterday));
-  }
+
+  // Determine date range from command line arguments, or default to yesterday --> today
+  // Note: The Profound API expects date inputs in EST, bc they execute prompts within a 24 hour EST window.
+  const today: Date = new Date();
+  const yesterday: Date = new Date();
+  yesterday.setDate(yesterday.getDate() - 1);
+  const start: string = startDateArg
+    ? startDateArg
+    : `${yesterday.getFullYear()}-${String(yesterday.getMonth() + 1).padStart(
+        2,
+        "0"
+      )}-${String(yesterday.getDate()).padStart(2, "0")}`;
```

Comment:
I think we could move this to a one-liner using the `Intl` API:

```typescript
const estDate = new Intl.DateTimeFormat("en-CA", {
  timeZone: "America/New_York",
  year: "numeric",
  month: "2-digit",
  day: "2-digit"
}).format(new Date());

console.log(estDate) // '2025-08-08'
```

(Note - The `en-CA` format makes it `YYYY-MM-DD` instead of `YYYY/MM/DD`)